www/src/components/email-capture-form.js

```diff
@@ -85,6 +85,65 @@ class EmailCaptureForm extends React.Component {
   }
 
   render() {
```

Comment:
Maybe the structure of this component would be simpler if it accepted `children`? The component can just handle the form based content, and text like:

> Enjoyed this post? Receive the next one in your inbox!

or 

> Sign up for the Gatsby newsletter.... 

can be child components. So you could use it like:

```jsx
  <EmailCaptureForm>
    <p>Enjoyed this post? Receive the next one in your inbox!</p>
  </EmailCaptureForm>
```

 See [@fk's recent PR](https://github.com/gatsbyjs/gatsby/pull/6363/files) for an example.

Reply:
I see. That makes a lot of sense! I'll update that component and consuming components